http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterpSpec.scala

```diff
@@ -1295,6 +1300,49 @@ final class AppServiceInterpSpec extends AnyFlatSpec with LeonardoTestSuite with
     }
   }
 
+  it should "reject create SAS app request if it's not from AoU UI" in {
```

Comment:
What do you think about also having a happy path test to make sure that the request is accepted in case of an AOU UI label + high security group on the google project?